py/surveyqa/summary.py

```diff
@@ -395,6 +395,25 @@ def exptime_dgb(string, color):
     fig.legend.click_policy="hide"
     return fig
 
+def get_moonplot(exposures, width=500, height=300):
+    p = bk.figure(plot_width=width, plot_height=height, x_axis_label = "MOONFRAC", y_axis_label = "MOONALT")
+
+    keep = exposures['PROGRAM'] != 'CALIB'
+    exposures_nocalib = exposures[keep]
+    color_mapper = LinearColorMapper(palette="Magma256", low=np.min(exposures_nocalib["MOONSEP"]), high=np.max(exposures_nocalib["MOONSEP"]))
```

Comment:
Let's fix the low/high values to be 0-180 so that it has a consistent color mapping range regardless of the details of the input data.